src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface Figure {

```

Comment:
Figure better make an abstract class with one common field for `color`.
method `getArea()` better move to the interface AreaCalculator 
and `getValue ()` method to the Printable interface (or you can choose some other naming)